config/database.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   adapter: postgresql
   encoding: unicode
-  pool: <%= ENV.fetch('OFN_DB_POOL', 5) %>
+  pool: <%= ENV.fetch('OFN_DB_POOL', 10) %>
```

Comment:
I checked ofn-install and we don't seem to be configuring PostgerSQL's [max_connections](https://www.postgresql.org/docs/10/runtime-config-connection.html) so I assume all servers will have the default `max_connections = 100 `. We could discuss whether or not that's a good idea though, but it falls out of this PR.

Reply:
Ah I think this might have been an attempt at fixing those flaky `NetRead::Timeout` errors in the CI build, but that's now been resolved in master with a different fix. I can't quite remember though. I could drop this commit and see what the build looks like?

Reply:
if this change is not required by Rails 5, I'd remove it, yes.

Reply:
Dropped :+1: Lets see if any phantom errors pop up in the build...